Exceptions should be handled, empty rescue blocks will supress it. If you are really confident that you can ignore the exception and your program will keep running, you can disable this warning.

[Source](http://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Lint/HandleExceptions)